### DISCUSSION TITLE
Vercel, Cyclic and Render are not connecting with GitHub repositories.

### DISCUSSION THREAD

**Discussion by techemmy** [community/community#35461] [Repositories] [answered]
I keep getting this error.
<img width="707" alt="Screenshot 2022-10-07 at 18 12 13" src="https://user-images.githubusercontent.com/43725109/194613333-ecfff404-37aa-4b23-a340-a91211d914d1.png">

**Comment by doguedogue** [comment]
where is this page? I like this angry unicorn 🦄😡

**Reply by techemmy** [reply]
It shows when you're trying to connect to a GitHub repo via Vercel, Render and Cyclic. I think it's a bug.

**Comment by jsoref** [comment]
When you persistently encounter a 🦄 , you should open a ticket with https://support.github.com

**Reply by techemmy** [reply]
Okay. Thanks, @jsoref.

**Comment by techemmy** [comment, marked as answer]
I've found a solution here
https://github.com/orgs/community/discussions/34751#discussioncomment-3797859
